BE/app/worker.py

```
@@ -268,3 +268,69 @@ def task_generate_script(self, topic: str, channel_profile: dict, topic_request_
+    매일 스케줄로 실행되어 등록된 모든 경쟁 채널의 최신 영상 3개를 가져옴
+    """
+    try:
+        logger.info(f"[Task {self.request.id}] 경쟁 유튜버 최신 영상 업데이트 시작")
+
+        # 비동기 함수 실행
+        result = asyncio.get_event_loop().run_until_complete(
+            _update_all_competitor_videos_async()
+        )
+
+        logger.info(f"[Task {self.request.id}] 경쟁 유튜버 최신 영상 업데이트 완료: {result}")
+        return result
```

Ruff (0.14.14)

[warning] Consider moving this statement to an `else` block

(TRY300)